neo/NeoSystem.cs

```diff
@@ -13,6 +13,9 @@ namespace Neo
 {
     public class NeoSystem : IDisposable
     {
+        private Peer.Start start_message = null;
+        private bool suspend = false;
```

Comment:
A better name would be `delay` or `shouldDelayStart`. Suspend implies that the node was running and it is being stopped and then started again.

Reply:
Actually the naming isn't that big of an issue, since the system has already been created, it could still be considered suspending.